pyroengine/engine/pyronear_engine.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+from .pyronear_predictor import PyronearPredictor
+from pyroclient import client
+
+
+api_url = "http://pyronear-api.herokuapp.com"
```

Comment:
I think this needs to be loaded from environment variable using a dotenv file for security, and maintenance